__tests__/app.test.js

```diff
@@ -237,6 +237,65 @@ describe.only("POST /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH:200 responds with the updated article", () => {
+    const update = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.votes).toBe(110);
+        expect(typeof article.author).toBe("string");
```

Comment:
You already have a test for the shape of an article - this test should only care about the votes being updated.